peer_mentoring/posts/models.py

```diff
@@ -15,5 +16,7 @@ class Post(models.Model):
     author = models.ForeignKey(User, on_delete=models.CASCADE)
     title = models.CharField(max_length=150)
     content = models.TextField(blank=False, null=True)
-    comments = models.ManyToManyField(Comments)
+    comments = models.ManyToManyField(Comment)
     added = models.DateTimeField(auto_now_add=True, null=False, blank=False)
+    is_group_post = models.BooleanField(blank=False, null=False, default=True)
+    group_id = models.ForeignKey(Group, on_delete=models.CASCADE, blank=True, null=True)
```

Comment:
I usually call my foreign keys an object so `group` in this case

Reply:
Changed the field name

Reply:
cool, btw what happened with the commit messages?

<img width="1134" alt="image" src="https://user-images.githubusercontent.com/387927/226932925-10473168-3847-4afc-ae42-4653dc2db93b.png">

Reply:
Was that me recommitting based on flake8/black fails? If not, sorry on that.

Reply:
no worries, was just wondering why the commit messages were all the same + it was not allauth related